models.py

```diff
@@ -0,0 +1,30 @@
+from sqlalchemy import Column, Integer, String, Date, ForeignKey
+from sqlalchemy.orm import relationship
+
+from database import BaseClass
```

Comment:
The import statement `from database import BaseClass` should be verified to ensure that `BaseClass` is correctly defined in `database.py` as the declarative base for SQLAlchemy models. This is crucial for the models to function properly.